### ISSUE TITLE
A.10.IR01.DQ_DomainConsistency.report.for.classification

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-interoperable-sds#16]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The following Note is, though formally correct, quite an overkill. Why can't people just declare the highest conformance class?
It's assumed, that separate gmd:report elements are provided for each passed classification invocable, interoperable and harmonised that the service conforms to. Otherwise the tests for these "super class" CCes will not pass.

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
